includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -143,6 +146,40 @@ public function add_product_bundles_classes ( $classes, $document_type, $order,
 		return $classes;
 	}
 
+
+	/**
+	 * WPC Product Bundles
+	 * @param string $classes       CSS classes for item row (tr) 
+	 * @param string $document_type PDF Document type
+	 * @param object $order         WC_Order order
+	 * @param int    $item_id       WooCommerce Item ID
+	 */
+	public function add_wpc_product_bundles_classes ( $classes, $document_type, $order, $item_id = '' ) {
+		if ( !class_exists('WPCleverWoosb') ) {
```

Comment:
I know you've adapted this from existing code, but we try to apply [WPCS](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/) on all new code as much as possible. In this case, that means adding a few extra spaces:
```php
if ( ! class_exists( 'WPCleverWoosb' ) ) {
```